scripts/start_med_cat_trainer.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+cd /home/ubuntu/MedCATtrainer || exit
+
+systemctl stop nginx.service
```

Comment:
Is the Nginx service expected to be always running? Just checking if this command would raise an error when running the script on a new instance from scratch. Maybe we need to add something to continue the script if that particular step fails.

Reply:
In our case there is no error. There was an issue where the base install of Nginx was blocking the Nginx container from starting correctly. I have removed nginx for the VM but some reason or another it was still blocking. Stopping service means the dockerise nginx can run correctly.

Reply:
Just checked running the command is not a problem for our script